script.js

```diff
@@ -0,0 +1,37 @@
+const header = document.querySelector('header');
+const section = document.createElement('section');
+header.appendChild(section);
+// section.innerHTML = `<p>Hello eorld</p>`;
+
+const open = document.querySelector('#menue-button');
+
+open.addEventListener('click', () => {
+  const window = document.createElement('div');
+  window.className = 'window';
+  const popup = document.createElement('div');
+  popup.className = 'popup';
+  popup.innerHTML = `<div class="all">
+  <i class="fa-solid fa-xmark close"></i>
+<ul id="popup">
+<li href="#portifolio">portifolio</li>
+<li>about</li>
+<li>contact</li>
+</ul>
+  </div>`;
+  window.appendChild(popup);
+  header.appendChild(window);
+
+  const close = document.querySelector('.close');
+  close.addEventListener('click', () => {
+    header.removeChild(window);
+  });
+  document.querySelector('#portifolio').addEventListener('click', () => {
+    window.removeChild(popup);
+  });
+  document.querySelector('#about').addEventListener('click', () => {
+    window.removeChild(popup);
+  });
+  document.querySelector('#contact').addEventListener('click', () => {
+    window.removeChild(popup);
```

Comment:
- [ ] When I click on any nav link it supposes to redirect me to the right sect on the page and closes the mobile menu that doesn't happen here in your app So please fix that.